src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -73,23 +84,36 @@ struct cbuf_info {
 struct cbuf_meta_range {
 	struct cbuf_meta *m;
 	vaddr_t dest;
-	u32_t low_id;
+	int low_id;
 	struct cbuf_meta_range *next, *prev;
 };
-#define CBUF_META_RANGE_HIGH(cmr) (cmr->low_id + (PAGE_SIZE/sizeof(struct cbuf_meta)))
+#define CBUF_META_RANGE_HIGH(cmr) (cmr->low_id + (int)(PAGE_SIZE/sizeof(struct cbuf_meta)))
 
 struct cbuf_bin {
-	int size;
+	unsigned long size;
 	struct cbuf_info *c;
 };
 
+struct blocked_thd {
+	unsigned short int thd_id;
+	unsigned long request_size;
+	struct blocked_thd *next, *prev;
+	unsigned long long blk_start;
+};
+
 struct cbuf_comp_info {
 	spdid_t spdid;
 	struct cbuf_shared_page *csp;
 	vaddr_t dest_csp;
 	int nbin;
 	struct cbuf_bin cbufs[CBUF_MAX_NSZ];
 	struct cbuf_meta_range *cbuf_metas;
+	unsigned long target_size, allocated_size;
```

Comment:
This may be me, but I don't like to put multiple fields in the same row in a struct definition. I don't know if this is covered or not in the style guide.